backend/src/DataAccessLayer/dalLoadClients.cs

```diff
@@ -49,7 +48,7 @@ public async Task<IEnumerable<LoadClient>> GetLoadClientsAsync()
             QueryDefinition sql = new QueryDefinition("select * from loadClients");
 
             // run query
-            FeedIterator<LoadClient> query = cosmosDetails.Container.GetItemQueryIterator<LoadClient>(sql);
+            FeedIterator<LoadClient> query = cosmosDetails.SourceContainer.GetItemQueryIterator<LoadClient>(sql);
 
```

Comment:
@joheec, let's sync on LoadClient.  I don't think we're storing it a an individual object in the CosmosDB.  I think we only have it as parts of other objects like status, test, and results.

Reply:
Agreed. This is just updating existing code with the new container name until the data access layer repo is set up so this can be ripped out and reorganize the api endpoints.

Reply:
I'd rather remove it since we know we're not persisting LoadClient objects directly to the store.